PowerShellGet/PSModule.psm1

```diff
@@ -13803,6 +13887,61 @@ function Get-PrivateData
     }
 }
 
+function Validate-PreReleaseString 
+{
+    [CmdletBinding(PositionalBinding=$false)]

```

Comment:
I assume this is an internal helper function. If this is the case, I recommend not to use CmdletBinding semantics.